user-operator/src/main/java/io/strimzi/operator/user/operator/KafkaUserQuotasOperator.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.operator.user.operator;
+
+import io.strimzi.operator.common.operator.resource.ReconcileResult;
+import io.vertx.core.Future;
+import io.vertx.core.Vertx;
+import io.vertx.core.json.JsonObject;
+
+public class KafkaUserQuotasOperator {
+
+    private KafkaUserQuotas quotasManager;
+
+    private Vertx vertx;
+
+    public KafkaUserQuotasOperator(Vertx vertx, KafkaUserQuotas quotasManager) {
+        this.quotasManager = quotasManager;
+        this.vertx = vertx;
+    }
+
+    Future<ReconcileResult<Void>> reconcile(String username, JsonObject quotas) {
+        Future<ReconcileResult<Void>> fut = Future.future();
+        vertx.createSharedWorkerExecutor("kubernetes-ops-pool").executeBlocking(
+            future -> {
+                boolean exists = quotasManager.exists(username);
```

Comment:
Since you don't use this in the `quotas != null` branch you should move it to the `else` branch.

Reply:
I use this at line 30.